airflow/contrib/plugins/metastore_browser/templates/metastore_browser/base.html

```diff
@@ -31,14 +31,14 @@ <h3 style="float: left">
 {% block head %}
 {{ super() }}
 <link rel="stylesheet" type="text/css" href="{{ url_for("static", filename="dataTables.bootstrap.css") }}">
-<link href="/admin/static/vendor/select2/select2.css" rel="stylesheet">
+<link href="{{ url_for('admin.index')}}static/vendor/select2/select2.css" rel="stylesheet">
```

Comment:
Should this, (all many similar url_for calls in the templates) not be `{{ url_for("static", filename="vendor/select2/select2.css") }}`?

Reply:
probably, but can I open a ticket on JIRA for someone to take it later? I may not have time to do it on this pull request

Reply:
+1 to @ashb.. in the current form the code doesn't look good..

Reply:
It is actually harder than I though, I'll change the ones that are easy but some are tricky (ex: task_instances)

Reply:
When I try `{{ url_for('admin.task_instances') }}` Flask throws a helpful error:

> werkzeug.routing.BuildError: Could not build url for endpoint 'task_instances'. Did you mean 'airflow.task_instances' instead?

And trying that we get the right URL: "/admin/airflow/object/task_instances"